src/sagas/projects.js

```diff
@@ -81,23 +84,20 @@ export function* importGist({payload: {gistId}}) {
 }
 
 export function* updateProjectSource() {
-  const state = yield select();
-  yield call(saveCurrentProject, state);
+  yield* saveCurrentProject();
 }
 
 export function* userAuthenticated() {
   const state = yield select();
-  yield fork(saveCurrentProject, state);
+  yield* fork(saveCurrentProject);
```

Comment:
As I understand it, `yield*` hands off control of the generator to another generator, but the `fork` effect is not itself a generator, so I don’t think it does anything here. For clarity I believe it should be changed back to a simple `yield`.